### PR TITLE
Add functions to `Package` to drain, move and remove files after the split

### PR DESCRIPTION
**Pull request by Arignir**

### CODE REVIEW

File: stdlib/package.py [hunk 1]
```diff
@@ -137,6 +139,176 @@ def __init__(
             shutil.rmtree(self.package_cache)
         os.makedirs(self.package_cache)
 
+    def is_empty(self) -> bool:
+        """Tests if the ``wrap_cache`` of this :py:class:`.Package` contains at least a single file.
```
Comment (Pedrochu) [inline stdlib/package.py]: Shouldn't this be an infinitive verb?
```suggestion
        """Test if the ``wrap_cache`` of this :py:class:`.Package` contains at least a single file.
```

File: stdlib/package.py [hunk 2]
```diff
@@ -137,6 +139,176 @@ def __init__(
...
+
+        :returns: ``True`` if the cache contains at least a single file, ``False`` otherwise.
+        """
+        for _, _, filenames in os.walk(self.wrap_cache):
+            if len(filenames):
+                return False
+        return True
+
+    def drain(self, *paths: str):
+        """Drain the current :py:class:`.Build`, moving files from its ``install_cache`` to this
```
Comment (Pedrochu) [inline stdlib/package.py]:
```suggestion
        """Drain the current :py:class:`.Build`, moving files from their ``install_cache`` to this
```
Same error for the next two functions, on the lines 195 and 234

File: stdlib/package.py [hunk 3]
```diff
@@ -137,6 +139,176 @@ def __init__(
...
+
+            lib_pkg.drain_package(
+                bin_pkg,
+                'usr/lib{32,64}/',
+                'usr/lib/lib*.so',
+            )
+
+        :note: Files are moved and not copied
+
+        :param source: The source package containg the files to drain from.
```
Comment (Pedrochu) [inline stdlib/package.py]:
```suggestion
        :param source: The source package containing the files to drain from.
```

File: stdlib/package.py [hunk 4]
```diff
@@ -137,6 +139,176 @@ def __init__(
...
+        :param src: A path pointing to the files to move, relative to the ``build_cache`` of the current :py:class:`.Build`,
+            This argument supports the globbing and braces syntax of common shells.
+        :param dst: A path pointing to the directory where the files should be moved, relative to the ``wrap_cache`` of this :py:class:`.Package`,
+
+        """
+        build = stdlib.build.current_build()
+
+        if os.path.isabs(src):
+            raise RuntimeError("Package.drain_build_cache() received an absolute path as parameter, but it expects a relative one")
+        if os.path.isabs(dst):
```
Comment (Pedrochu) [inline stdlib/package.py]: This condition could be also put in the same `if` as the one just above, although it's not that big of a deal. The same thing happens in the next function

**Review by Pedrochu — DISMISSED** [review]

File: stdlib/package.py (hunk 4 above)
Comment (Arignir) [reply stdlib/package.py]: Yep you're right!

File: stdlib/package.py (hunk 1 above)
Comment (Arignir) [reply stdlib/package.py]: It should, thanks!

Comment (doom) [inline stdlib/package.py]:
```suggestion
        """Test whether the ``wrap_cache`` of this :py:class:`.Package` contains at least a single file.
```

File: stdlib/package.py [hunk 5]
```diff
@@ -137,6 +139,176 @@ def __init__(
...
+
+                    os.makedirs(os.path.dirname(dstpath), exist_ok=True)  # Create parent directories (if any)
+                    shutil.move(rpath, dstpath)
+
+    def move(self, srcs: str, dst: str):
+        """Move the files pointed to by ``srcs`` to ``dst``.
+
+        :param srcs: The paths pointing to the files to move, relative to the ``wrap_cache`` of this :py:class:`.Package`.
+            This argument supports the globbing and braces syntax of common shells.
+        :param dst: A path pointing to the destination folder, relative to the ``wrap_cache`` of this :py:class:`.Package`.
```
Comment (doom) [inline stdlib/package.py:276]: You might want to assert that `dst` is indeed a directory

**Review by doom — APPROVED** [review]

**Pull request merged**